notes/push/base.sh

```diff
@@ -0,0 +1,231 @@
+#|     return await commands.push(
+#|            ^^^^^^^^^^^^^^^^^^^^
+#|   File "spinta/commands/write.py", line 96, in push
+#|     status_code, response = await _batch_response(
+#|                             ^^^^^^^^^^^^^^^^^^^^^^
+#|   File "spinta/commands/write.py", line 1162, in _batch_response
+#|     async for data in results:
+#|   File "spinta/accesslog/__init__.py", line 174, in log_async_response
+#|     async for row in rows:
+#|   File "spinta/commands/write.py", line 187, in push_stream
+#|     async for data in dstream:
+#|   File "spinta/backends/postgresql/commands/changes.py", line 25, in create_changelog_entry
+#|     async for data in dstream:
+#|   File "spinta/backends/postgresql/commands/write.py", line 31, in insert
+#|     async for data in dstream:
+#|   File "spinta/commands/write.py", line 861, in prepare_data_for_write
+#|     async for data in dstream:
+#|   File "spinta/commands/write.py", line 648, in prepare_patch
+#|     async for data in dstream:
+#|   File "spinta/commands/write.py", line 618, in validate_data
+#|     async for data in dstream:
+#|   File "spinta/commands/write.py", line 521, in read_existing_data
+#|     async for data in dstream:
+#|   File "spinta/commands/write.py", line 510, in prepare_data
+#|     commands.simple_data_check(context, data, data.model, data.model.backend)
+#|   File "multipledispatch/dispatcher.py", line 278, in __call__
+#|     return func(*args, **kwargs)
+#|            ^^^^^^^^^^^^^^^^^^^^^
+#|   File "spinta/backends/__init__.py", line 150, in simple_data_check
+#|     simple_model_properties_check(context, data, model, backend)
+#|   File "spinta/backends/__init__.py", line 177, in simple_model_properties_check
+#|     simple_data_check(
+#|   File "spinta/backends/__init__.py", line 290, in simple_data_check
+#|     raise exceptions.NotImplementedFeature(prop, feature="Ability to indirectly modify base parameters")
+#| spinta.exceptions.NotImplementedFeature: Ability to indirectly modify base parameters is not implemented yet.
+#|   Context:
+#|     component: spinta.components.Property
+#|     manifest: default
+#|     schema: 11
+#|     dataset: push/base
+#|     resource: db
+#|     model: push/base/City
+#|     entity: cities
+#|     property: name
+#|     attribute: name
+#|     resource.backend: push/base/db
+#|     feature: Ability to indirectly modify base parameters
+#| 
+#| INFO:     127.0.0.1:41448 - "POST / HTTP/1.1" 500 Internal Server Error
+# FIXME: It looks, that there is a multiple errors here.
+#        1. It looks, that push attempts to write to base (Location) model, by
+#           sending `name` which is part of Location, and not part of City.
+#           I think, that push should only send data, that directly belongs to
+#           model and do not send properties, that comes from base.
+#        2. push command couldn't correctly handle error received from remote
+#           server and failed, when writing to push state table.
```

Comment:
It seems, that push with base does not work, probably main issue, that push sends properties, that do not belong to model directly. Maybe fixing this, push starts to work?